shared/js/ui/views/site.es6.js

```diff
@@ -174,15 +179,17 @@ Site.prototype = window.$.extend({},
             })
         },
 
-        _showPageTrackers: function () {
+        _showPageTrackers: function (e) {
+            if (e) e.preventDefault()
```

Comment:
Added these here because clicking on the actual links was throwing errors in the extension popup console.